### ISSUE TITLE
Some questions about the pretraining setting

### ISSUE THREAD

**Issue by LenarBe** [wtl52656/TransferTraj#2]
Thanks you @wtl52656 for your impressive work! However I have some questions about the setting. Firstly, the code released uses Chengdu as the pretraining and finetuning dataset at the same time, and the pretraining function is almost the same as the finetuning. So is the pretraining setting in the paper actually uses the same dataset as the evaluation, or is there another synthetic pretraining dataset utilized? And in this circumstances, what's the main difference between pretraining and finetuning? Thank you for your reading and I'm looking forward for your reply.

**Comment by wtl52656** [comment]
Thanks for your attention. Pretraining and fine-tuning can be performed on the same dataset. The difference in their settings lies in the **"padder"** part. In pretraining, the trajectory points or temporal/spatial modalities to be masked are randomly selected, while in finetune, "padder" uses different masking mechanisms based on the task.
